### ISSUE TITLE
Интерфейс. Предложения по оптимизации

### ISSUE THREAD

**Issue by Baron-Munchhausen** [dim5x/PyNetHomeInvaderAlerter#40]
Есть предложение по изменению интерфейса

**Comment by dim5x** [comment]
![image](https://user-images.githubusercontent.com/49735406/90497688-070ac100-e150-11ea-9a27-abd44c8783d7.png)

> при выборе mac'а в current_state лог должен фильтроваться по этому mac'у.
Сейчас у нас при выборе мака срабатывает механизм добавления в доверенные\недоверенные. Как ты видишь реализацию ещё и фильтрации по маку?

**Comment by dim5x** [comment]
> чек-бокс "Только недоверенные mac'и" (или пусть верхний чек-бокс так же управляет и логом)

А сортировка по поиску сейчас это не то?
![image](https://user-images.githubusercontent.com/49735406/90498928-9369b380-e151-11ea-8fcc-edc38f91c3ea.png)

![image](https://user-images.githubusercontent.com/49735406/90498878-864cc480-e151-11ea-9004-176084206076.png)

**Comment by Baron-Munchhausen** [comment]
Согласен: фильтры по железке и по типу mac'а избыточны, достаточно поиска.
Добавил колонку MacType в таблицу состояний.
Предлагаю так:
- если кликнул на тип mac'а, то переход на изменение статуса "доверенный/недоверенный"
- если кликнул на сам mac, то тогда фильтр лога. Можно сделать автоматическое заполнение поискового поля?

И тогда последнее осталось: "фильтр по периоду".

Предлагаю убрать чек-бокс "лог". Пусть панель будет постоянной.

**Comment by dim5x** [comment]
> если кликнул на сам mac, то тогда фильтр лога. Можно сделать автоматическое заполнение поискового поля?

Добавил вставку мака в поле поиска, но так как это не форма, у поля нет функции .submit() - чтобы эмулировать нажатие клавиши ввод. Сейчас, после того как перенесся мак в поле поиска, нужно кликнуть куда-нибудь в поле и поиск сработает.
Пока не знаю как это можно обойти, чтобы и от этого нажатия избавиться...

**Comment by dim5x** [comment]
Научился переводить фокус на поле поиска - ещё на полшага к полноценному клику!

Сейчас можно выделять необходимый мак и жать энтер для поиска.
